debian10/metadata_summary.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file also doesn't need to be created manually and will be generated by the first DUS run.